### ISSUE TITLE
Erroneously marks 4 space indentations as warning for unordered lists

### ISSUE THREAD

**Issue by poshcodebear** [DavidAnson/vscode-markdownlint#50]
When working with unordered lists, if I'm using 4 space indentations (as per the spec), it marks them as invalid with ```[markdownlint] MD007/ul-indent: Unordered list indentation [Expected: 2; Actual: 4]```.

Example:

```markdown
- Base item
    - Indented item
```
Ordered lists, on the other hand, don't get warnings for either 2 or 4 space indentations.

Since 4 spaces is what's specified in the official spec, and it's more that many implementations let you do 2 spaces, I feel this should be corrected to allow 4 spaces without warnings.

**Comment by DavidAnson** [comment]
I don’t think it’s quite true that the spec calls for 4 spaces. For that and more discussion of this topic, please see https://github.com/DavidAnson/markdownlint/issues/141 and linked issues.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by poshcodebear** [comment]
It occurs to me that, when searching to see if this was open, I should have also looked for MD007.

Anyway, thanks for pointing that out!  I thought the original design spec said 4, but I'll be the first to point out that I can make mistakes :)

Anyway, if there's any way for it to support both, that would be great; maybe have the rule, instead of "x number of spaces on indent" be something like "consistent indentation size through the document"?  I have no idea how doable that is or how much processing that would add to the extension, though.

You mentioned in the other thread that this behavior is configurable; I hunted that down and set MD007 to 4 spaces in my VSCode user settings and that seems to have done the trick for me.  Maybe a solution could be adding to the tool-tip somehow to hint that it's configurable, maybe with a link to instructions for that?

Anyway, thank you for this extension, it has helped me get a lot better in my markdown :)

**Comment by DavidAnson** [comment]
Cool, thanks!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
